.eslintignore

```diff
@@ -64,7 +77,5 @@ xmodule/js/spec/problem/edit_spec_hint.js
 xmodule/js/spec/problem/edit_spec.js
 xmodule/js/spec/tabs/edit.js
 
-xmodule/js/public/js
-xmodule/assets/*/public/js
```

Comment:
Reviewers:

xmodule/js/public/js was moved up with the rest of the symlinks.

xmodule/assets used to contain symlinks, but no longer does, so it can be un-ignored.